shared/chat/conversation/messages/text/container.js

```diff
@@ -1,22 +1,47 @@
 // @flow
 import * as Constants from '../../../../constants/chat2'
 import * as Types from '../../../../constants/types/chat2'
+import * as Chat2Gen from '../../../../actions/chat2-gen'
 import TextMessage from '.'
 import {namedConnect} from '../../../../util/container'
 
 type OwnProps = {|
   message: Types.MessageText,
 |}
 
+const getReplyProps = (replyTo, onReplyClick) => {
+  if (!replyTo || replyTo.type !== 'text') {
+    return undefined
+  }
+  return {
+    edited: replyTo.hasBeenEdited,
+    onClick: () => onReplyClick(replyTo.id),
+    text: replyTo.text.stringValue(),
+    username: replyTo.author,
+  }
+}
+
 const mapStateToProps = (state, ownProps: OwnProps) => {
   const editInfo = Constants.getEditInfo(state, ownProps.message.conversationIDKey)
   const isEditing = !!(editInfo && editInfo.ordinal === ownProps.message.ordinal)
   return {isEditing}
 }
-const mapDispatchToProps = dispatch => ({})
+
+const mapDispatchToProps = (dispatch, {message}: OwnProps) => ({
+  _onReplyClick: messageID =>
+    dispatch(
+      Chat2Gen.createLoadMessagesCentered({
```

Comment:
Could this clear out the `centeredOrdinalInfo` for this convo? I think that'd make it so messages can flash more than once per mount

Reply:
Done